plugins/inputs/lanz/lanz.go

```diff
@@ -130,3 +120,13 @@ func msgToAccumulator(acc telegraf.Accumulator, msg *pb.LanzRecord, deviceURL *u
 		acc.AddFields("lanz_global_buffer_usage_record", vals, tags)
 	}
 }
+
+func newLanz() *Lanz {
+	return &Lanz{}
+}
```

Comment:
Do you think we can get rid of this function in the present PR?

Reply:
Removed.